build/publish/Microsoft.DotNet.Cli.Checksum.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="GenerateChecksums"
+          DependsOnTargets="Init;
+                            SetupGenerateChecksumsInputs;"
+          Inputs="@(GenerateChecksumsInputs)"
+          Outputs="%(GenerateChecksumsInputs.DestinationPath)">
+    <GenerateChecksums Items="@(GenerateChecksumsInputs)" />
```

Comment:
I've renamed it to `ArtifactsForGeneratingChecksums`, and renamed the target which initializes it to `InitializeChecksumItemGroup`.